tests/libusb_testlib.h

```diff
@@ -20,7 +20,8 @@
 #ifndef LIBUSB_TESTLIB_H
 #define LIBUSB_TESTLIB_H
 
-#include <config.h>
+#include "config.h"
+#include "libusbi.h"
```

Comment:
I guess this is the disadvantage of not using m4 (and config.h), now "user" programs need to include this internal header file.